src/UserService.Data/UserRepository.cs

```diff
@@ -195,6 +204,10 @@ public bool SwitchActiveStatus(Guid userId, bool status)
             dbUser.IsActive = status;
 
             _provider.Users.Update(dbUser);
+            dbUser.ModifiedBy = _httpContextAccessor.HttpContext.Items.ContainsKey(ConstStrings.UserId) ?
```

Comment:
strange. how i remember it create user, so userId have to be in context, also in next line you use ?. before HttpContext, but here not, so if httpContext really null you will have HullRefExc before checking

Reply:
you are right, without "?"